starters/academic/config/_default/params.yaml

```diff
@@ -81,6 +81,14 @@ features:
       show_count: true
     commento:
       url: ''
+    giscus:
```

Comment:
Please just list the required params below in each Params file as we don't want this file getting too bloated. If users want to see every option, they can find the other options in the docs.

```
      repo: ''
      repo_id: ''
      category: ''
      category_id: ''
```

Reply:
Okay. I've made the changes.